buddyboss/buddyboss-platform
# PROD-709 Network search result count issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3094  .

### How to test the changes in this Pull Request:

1. "Blog pages show at most" set to 5 from Settings > Reading page.
2. Go to frontend and do a network search with a keyword that has more then 5 blog posts.
3. Click on the Blog Posts from left content category panel.
4. See it works fine now.

### Proof Screenshots or Video

https://watch.screencastify.com/v/dRvW1KYPo1qcHgy0slvK

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
